src/views/htmlcontent/tsconfig.json

```diff
@@ -7,6 +7,7 @@
         "sourceMap": true,
         "rootDir": ".",
         "target": "ES6",
-        "types": []
+        "types": [],
+        "noUnusedLocals": true
```

Comment:
I'm not really a fan of this approach, since it means making everything protected in angular code, I would rather move to using ESLint over TSLint and keep it has a linting rule that can be disabled with comments. https://github.com/palantir/tslint/issues/4534

Reply:
Hm, that definitely seems like the right long-term approach but I don't think I can prioritize that work over the other stuff I'm assigned to right now. How about as a temporary fix I remove this option and revert the private -> protected changes then create a follow-up issue to move over to ESLint that me or someone else can do once we have time? That way @shengyfu can get unblocked with his Intellicode work.